helm-charts/splunk-otel-collector/values.schema.json

```diff
@@ -1,1410 +1,1557 @@
 {
-  "$schema": "http://json-schema.org/schema#",
```

Comment:
Do you mind providing only the minimum required changes here? It looks like some descriptions are getting removed.